.solhint.json

```diff
@@ -9,10 +9,9 @@
     "avoid-throw": "error",
     "avoid-tx-origin": "off",
     "check-send-result": "error",
-    "compiler-version": ["error", "0.6.6"],
+    "compiler-version": "off",
     "const-name-snakecase": "error",
     "func-name-mixedcase": "error",
-    "func-order": "error",
```

Comment:
is this removed?

Reply:
Accidentally removed it when testing the linting. Returned now.